bigcache_test.go

```diff
@@ -617,6 +617,21 @@ func TestNilValueCaching(t *testing.T) {
 	assert.Equal(t, []byte{}, cachedValue)
 }
 
+func TestClosing(t *testing.T) {
```

Comment:
I think in this test we would like to track number of goroutines, possibly via [`runtime.NumGoroutine`](https://golang.org/pkg/runtime/#NumGoroutine)

Reply:
+1 from me.

Reply:
I thought about this but this would require not running this test in parallel. I've also seen tests that rely on these numbers to be flaky. I'm more than happy to do it though, just a heads up.

Reply:
We'll start here and see how it goes. :)